frontend/src/features/projects/dispose/ProjectDisposeLayout.tsx

```diff
@@ -1,7 +1,7 @@
-import React, { useEffect, useRef, useState } from 'react';
```

Comment:
Thanks for this! Was just working on this on my end and cleaning it up then had a flashback to reviewing this yesterday 🤦 

I was cringing at my old approach while fixing it lol got rid of all the preDraft stuff and the extra form.  But I was stuck on the point where the next button would work; however, it would not mark step one as complete. But this looks like it covers that!!